### ISSUE TITLE
Null results for Closeness and Betweenness

### ISSUE THREAD

**Issue by caitiewrites** [neo4j-contrib/neo4j-graph-algorithms#413]
```
CALL algo.closeness.stream('User') YIELD nodeId, centrality WHERE centrality is not null
WITH * ORDER BY centrality DESC LIMIT 5
RETURN nodeId, centrality;
```
Returns 
<img width="875" alt="screen shot 2017-10-04 at 17 03 54" src="https://user-images.githubusercontent.com/8246056/31186782-92075e30-a927-11e7-9318-9139d639bf3c.png">

Schema for reference
<img width="1367" alt="screen shot 2017-10-04 at 16 33 00" src="https://user-images.githubusercontent.com/8246056/31186883-df7bf0f4-a927-11e7-9a9b-6ca1bc999dfb.png">

For reference we use the Thread nodes to create the TALKS_TO relationship, then we drop the Thread nodes.  That's why it's not connected to the User nodes.

I have also tried the Betweenness algorithm with similar null results.  I've successfully gotten results from PageRank, though.

**Comment by jexp** [comment]
Oh that is the issue we had before. It is `Infinity`, only that JS renders it as null.

Do you build it yourself locally? What Neo4j version are you using? We have a new release upcoming, that might fix it.

https://github.com/neo4j-contrib/neo4j-graph-algorithms/issues/256

**Comment by caitiewrites** [comment]
Thanks @jexp .  We're using version 2.0.0-M10 .  I'm confused about why the relationships would be infinity, but perhaps that is something related to our particular data set.

**Comment by jexp** [comment]
What version is 2.0.0-M10 that's not a neo version nor one of the algorithms library ?

**Comment by caitiewrites** [comment]
<img width="241" alt="screen shot 2017-10-05 at 10 50 21" src="https://user-images.githubusercontent.com/8246056/31221272-03b14166-a9bb-11e7-9039-6a57f85fcb89.png">

Is this what you meant or were you asking about something else?

**Comment by mknblch** [comment]
The infinity issue of closeness centrality should have been fixed with #310

**Comment by mneedham** [comment]
Closing this as the infinity issue is addressed.
